src/payments/services/premium-certificate-purchase-email.service.ts

```
@@ -0,0 +1,165 @@
+      );
+
+      const errors = mailSend?.result?.email?.errors;
+      if (Array.isArray(errors) && errors.length > 0) {
+        this.logger.error(
+          `Premium certificate purchase email reported errors for intent ${params.paymentIntentId}: ${JSON.stringify(errors)}`,
+        );
+        return;
+      }
+
+      this.logger.log(
+        `Premium certificate purchase email sent for intent ${params.paymentIntentId} to ${user.email}`,
```

ESLint

[error] Delete `,`

(prettier/prettier)